assets/css/glimesh/variables.scss

```diff
@@ -122,6 +122,15 @@ $custom-select-focus-border-color: var(--input-border-color);
 $table-color:                 var(--body-color);
 $table-border-color:          var(--secondary-hover-bg-color);
 
+// Apex charts
```

Comment:
Can you put these styles in their own file under `css/glimesh/components` ?

Reply:
I moved them. `css/glimesh/components/apex-charts.scss`